libvast/include/vast/system/actors.hpp

```diff
@@ -468,6 +468,7 @@ CAF_ALLOW_UNSAFE_MESSAGE_TYPE(std::shared_ptr<vast_uuid_synopsis_map>)
 CAF_ALLOW_UNSAFE_MESSAGE_TYPE(vast::partition_synopsis_ptr)
 CAF_ALLOW_UNSAFE_MESSAGE_TYPE(vast::partition_synopsis_pair)
 CAF_ALLOW_UNSAFE_MESSAGE_TYPE(vast::pipeline_ptr)
+CAF_ALLOW_UNSAFE_MESSAGE_TYPE(std::shared_ptr<vast::pipeline>)
```

Comment:
Same as for the comment in fwd.hpp: IIUC this is temporary, so please add a comment to remind us to remove it again.